sdk/core/Azure.Core/src/Azure.Core.csproj

```diff
@@ -23,9 +23,16 @@
     <SupportedPlatform Include="browser" />
   </ItemGroup>
 
+  <!-- Keep System.Diagnostics.DiagnosticSource 5.0.0 for netcoreapp2.1 support. -->
+  <ItemGroup Condition=" '$(TargetFramework)' == 'netcoreapp2.1' ">
```

Comment:
Wouldn't we have to do something similar to any other SDKs that consume DiagnosticSource dependency directly? If so, should we not make the package.data.props have this conditional step as well?

Reply:
This is a good point. Right now, there are only a couple other libraries using this package, but it would probably break their netcoreapp2.1 builds.

Reply:
I believe most SDKs do not do multitargeting and are build for NetStandard so the chances of it failing for them would be very low. However a case can happen in future so I think we should do this for the sake of completeness.

Reply:
We talked about this in the core meeting and decided that we want to keep the multi-targeting scoped just to core for simplicity.